fluent-plugin-audit_log_parser.gemspec

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+Gem::Specification.new do |spec|
+  spec.name          = "fluent-plugin-audit_log_parser"
```

Comment:
Prefer single tick instead of quote unless doing string substitution like "my message #{@foo}"